Multisig/UI/Settings/App Settings/BasicAppSettingsView.swift

```diff
@@ -82,6 +62,33 @@ struct BasicAppSettingsView: View {
             self.trackEvent(.settingsApp)
         }
     }
+
+    @ViewBuilder
+    var signingWalletView: some View {
+        if signingKeyAddress != nil {
+            ForEach(0..<1) { _ in
```

Comment:
I'm not going to refactor  it now, but was curious why there's ForEach to produce just 1 element?

Reply:
to enable swipe to delete

Reply:
wow!